docs/Getting Started/Getting-Started-From-Scratch.md

```diff
@@ -1,100 +1,1127 @@
+    $ chmod +x startnode1.sh 
+    $ ./startnode1.sh
+    ``` 
+
+    Your node is now operational and you may attach to it with below commands`. This configuration starts Quorum without privacy support as could be evidenced in prefix `PRIVATE_CONFIG=ignore`, please see below sections on [how to enable privacy with privacy transaction managers](#adding-privacy-transaction-manager).
+
+    ```
+    $ geth attach new-node-1/geth.ipc
+    Welcome to the Geth JavaScript console!
+    
+    instance: Geth/v1.8.18-stable-bb88608c(quorum-v2.2.3)/darwin-amd64/go1.10.2
+    coinbase: 0xedf53f5bf40c99f48df184441137659aed899c48
+    at block: 0 (Thu, 01 Jan 1970 01:00:00 BST)
+     datadir: /Users/krish/fromscratch/new-node-1
+     modules: admin:1.0 debug:1.0 eth:1.0 ethash:1.0 miner:1.0 net:1.0 personal:1.0 raft:1.0 rpc:1.0 txpool:1.0 web3:1.0
+    
+    > raft.cluster
+    [{
+        ip: "127.0.0.1",
+        nodeId: "a5596803caebdc9c5326e1a0775563ad8e4aa14aa3530f0ae16d3fd8d7e48bc0b81342064e22094ab5d10303ab5721650af561f2bcdc54d705f8b6a8c07c94c3",
+        p2pPort: 21000,
+        raftId: 1,
+        raftPort: 50000
+    }]
+    > raft.leader
+    "a5596803caebdc9c5326e1a0775563ad8e4aa14aa3530f0ae16d3fd8d7e48bc0b81342064e22094ab5d10303ab5721650af561f2bcdc54d705f8b6a8c07c94c3"
+    > raft.role
+    "minter"
+    > 
+    > exit
+    ```
 
 ### Adding additional node
 1. Complete steps 1, 2, 5, and 6 from the previous guide
+
+    ```
+    $ mkdir new-node-2
+    $ bootnode --genkey=nodekey2
+    $ cp nodekey2 new-node-2/nodekey
+    $ bootnode --nodekey=new-node-2/nodekey --writeaddress
+    56e81550db3ccbfb5eb69c0cfe3f4a7135c931a1bae79ea69a1a1c6092cdcbea4c76a556c3af977756f95d8bf9d7b38ab50ae070da390d3abb3d7e773099c1a9
+    ```
 2. Retrieve current chains `genesis.json` and `static-nodes.json`. `static-nodes.json` should be placed into new nodes data dir
-3. Initialize new node with `geth --datadir new-node-2 init genesis.json`
-4. Edit `static-nodes.json` and add new entry for the new node you are configuring (should be last)
-5. Connect to an already running node of the chain and execute `raft.addPeer('enode://new-nodes-enode-address-from-step-6-of-the-above@127.0.0.1:21001?discport=0&raftport=50001')`
-6. Start your node and send into background with `PRIVATE_CONFIG=ignore nohup geth --datadir new-node-2 --nodiscover --verbosity 5 --networkid 31337 --raft --raftport 50001 --raftjoinexisting RAFT_ID --rpc --rpcaddr 0.0.0.0 --rpcport 22001 --rpcapi admin,db,eth,debug,miner,net,shh,txpool,personal,web3,quorum,raft --emitcheckpoints --port 21001 2>>node.log &`, where `RAFT_ID` is the response of the `raft.addPeer()` command in step 5.
+    ```
+    $ cp static-nodes.json new-node-2    
+    ```
+
+3. Edit `static-nodes.json` and add new entry for the new node you are configuring (should be last)
+   ```
+   $ vim static-nodes.json 
```

Comment:
It is not clear here whether you are editing the static-nodes.json from the new-node-2 directory or whether it is the one in the parent directory.

If it is the one in the parent directory - you should add another copy command - so that new-node-2 contains the updated file.